### DISCUSSION TITLE
Method to check container exists

### DISCUSSION THREAD

**Discussion by OpsOwns** [Azure/azure-cosmos-dotnet-v3#2554] [Q&A] [answered]
How can i check if container exists ?
 I want to check container exist before get data or do other stuff i recently found method like CreateContainerIfNotExistsAsync but it's not what i am looking for i tried GetContainer but i don't see any method or property.

**Comment by j82w** [comment, marked as answer]
Here is a sample:
https://github.com/Azure/azure-cosmos-dotnet-v3/blob/d3773388f54c996ef844d9a2b12afad37e436291/Microsoft.Azure.Cosmos.Samples/Usage/ContainerManagement/Program.cs#L226
